noggin/templates/_form_macros.html

```diff
@@ -39,3 +39,8 @@
     </div>
   {% endif %}
 {% endmacro %}
+
+{% macro unsaved_changes() %}
+<script src="{{ url_for('static', filename='js/vendor/jquery.dirty-0.7.2/jquery.dirty.js') }}"></script>
+<script>$("form").dirty({preventLeaving: true})</script>
```

Comment:
You may want to wrap that in `$(...)` to make sure it's executed after the document has loaded, no?

Reply:
Also, I'm thinking we may want that on all our forms, don't we? If so we can do it at the top-level template.

Reply:
There are some that we def dont want this on, like login, or register...